### DISCUSSION TITLE
Is there a possibility for Kalpa, Aeon and/or Leap Micro support?

### DISCUSSION THREAD

**Discussion by WesfunOfficial** [agama-project/agama#1686] [General]
I know all 3 of these projects plan on or are using different installers outside of yast-installer but would we be able to add them to agama at some point? I think it would be really nice to give users the options of all 7 distros but of course I'm unsure if its possible and if we could get support from Richard, Shawn, etc.

**Comment by jreidinger** [comment]
Technically nothing prevents using agama to install it, but of course question is if maintainers of given distros would support systems installed via agama and not their selected installer. And they will need to provide some settings for proposal like mandatory patterns or minimal requirements for given partitions.

**Comment by ancorgs** [comment]
Developers of Kalpa and Aeon explicitly stated they will not support installations done with Agama and even opened a pull request to remove the support that was originally there.

See https://github.com/agama-project/agama/pull/1243

So the answer to the original question is "no, there is no such a possibility for now.
